types/echarts/options/series/bar.d.ts

```diff
@@ -0,0 +1,16600 @@
+declare namespace echarts {
+    namespace EChartOption {
+        /**
+         * **bar chart**
+         *
+         * Bar chart shows different data through the height of a bar, which
+         * is used in
+         * [rectangular coordinate](https://ecomfe.github.io/echarts-doc/public/en/option.html#grid)
+         * with at least 1 category axis.
+         *
+         *
+         * @see https://ecomfe.github.io/echarts-doc/public/en/option.html#series-bar
+         */
+        interface SeriesBar {
+            [k: string]: any;
```

Comment:
What does this `k` mean?

Reply:
Sorry for that, it was a local trick for particular project. I'll remove it

Reply:
Done.